prow/pluginhelp/hook/hook.go

```diff
@@ -83,7 +83,12 @@ func (ha *HelpAgent) generateNormalPluginHelp(config *plugins.Configuration, rev
 			ha.log.Warnf("No help is provided for plugin %q.", name)
 			continue
 		}
-		help, err := provider(config, revMap[name])
+		var enabledRepos []plugins.Repo
+		for _, repo := range revMap[name] {
```

Comment:
Whats the reason for not using the `Repo` struct as values in the `revMap`?

Reply:
I just wanted to focus on the help provider in this PR. Then I will refine in specific areas like here and some parts of Config